lib/manager/azure-pipelines/index.ts

```diff
@@ -0,0 +1,5 @@
+export { extractPackageFile } from './extract';
+
+export const defaultConfig = {
+  fileMatch: ['azure.*pipelines?.*.ya?ml$'],
```

Comment:
```suggestion
  fileMatch: ['azure.*pipelines?.*\\.ya?ml$'],
```

Reply:
What about a starting optional dot?

Reply:
There's no `^` so the above will allow dot and more. e.g. could even be `config/azure-pipeline.yml`

Reply:
Yes, you are right. Have not seen it. 🙈